app/services/cql_query.rb

```diff
@@ -0,0 +1,21 @@
+class CqlQuery
+  attr_reader :args, :sortby
+
+  def initialize(sortby: nil, **args)
+    @args = args
+    @sortby = sortby
+  end
+
+  def to_query
+    [
+      args.map { |k, v| "#{k}==\"#{escape(v)}\"" }.join(' and '),
+      ("sortby #{Array(sortby).join(' ')}" if sortby)
+    ].compact.join(' ').strip
+  end
```

Comment:
could this just be `to_s`? or could we alias `to_s` to this? i wonder if that will make it not necessary to call explicitly in some places, like the call sites in `FolioClient` currently